sync_with_branch.sh

```diff
@@ -1,16 +1,22 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Please use `/bin/bash` to keep this working on NixOS.

Reply:
Fixed